install.sh

```diff
@@ -187,6 +206,29 @@ fi
 cd "/usr" || exit 1
 
 ####################################################################### script
+if [[ -n "${HOMEBREW_ON_LINUX-}" ]]; then
+  if [[ -n "${CI-}" ]] || [[ -w "$HOMEBREW_PREFIX_DEFAULT" ]] || [[ -w "/home/linuxbrew" ]] || [[ -w "/home" ]]; then
```

Comment:
What's this for?

Reply:
I have no idea, just ported it from `Linuxbrew/install`. @sjackman might know it.